Samples/PieChart/README.md

```diff
@@ -0,0 +1,27 @@
+# Pie Chart
+
+This example shows how you can create simple, but good looking graphs using the premium Fuse.Charting library. In this example, we will create a reusable line-graph renderer, which will be used to draw 3 similar graphs on top of each other.
```

Comment:
you need to update this text so that it matches the fact that you are making a pie chart:
> create a reusable line-graph renderer
for example